doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -138,6 +138,19 @@ each replica, including the times at which it was created and dropped
 | `dropped_at`          | [`timestamp with time zone`] | The time at which the replica was dropped, or `NULL` if it still exists.                                                                  |
 | `credits_per_hour`    | [`numeric`]                  | The number of compute credits consumed per hour. Corresponds to [`mz_cluster_replica_sizes.credits_per_hour`](#mz_cluster_replica_sizes). |
 
+### `mz_compute_dependencies`
+
+The `mz_compute_dependencies` table describes the dependency structure between each compute object (index, materialized view, or subscription) and the sources of its data.
+
+In contrast to [`mz_object_dependencies`](#mz_object_dependencies), this table only lists dependencies in the compute layer.
+SQL objects that don't exist in the compute layer (such as views) are omitted.
+
+<!-- RELATION_SPEC mz_internal.mz_compute_dependencies -->
+| Field       | Type     | Meaning                                                                                                                                                                                                                                                                                            |
+| ----------- | -------- | --------                                                                                                                                                                                                                                                                                           |
+| `object_id` | [`text`] | The ID of a compute object. Corresponds to [`mz_catalog.mz_indexes.id`](../mz_catalog#mz_indexes), [`mz_catalog.mz_materialized_views.id`](../mz_catalog#mz_materialized_views), or [`mz_internal.mz_subscriptions`](#mz_subscriptions).                                                           |
+| `import_id` | [`text`] | The ID of a compute import. Corresponds to [`mz_catalog.mz_indexes.id`](../mz_catalog#mz_indexes), [`mz_catalog.mz_materialized_views.id`](../mz_catalog#mz_materialized_views), [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources), or [`mz_catalog.mz_tables.id`](../mz_catalog#mz_tables). |
```

Comment:
It's an import to the dataflow. My thinking was that being more specific than "dependency" would be good to not make it seem like this table lists all things a compute object could possibly depend on (like the cluster, or view definitions). But happy to change this to `dependency_id` if people prefer this!

Reply:
Maybe we could define what `import` means here in the docs in that case? It seems like it has a specific meaning, so I don't want to change the name if this one is the most precise, but in that case we need to explain what this concept means.

Reply:
I think I'd prefer using a word that's immediately clear rather than adding another wall of text :) The "Meaning" column already lists all the things that imports/dependencies can be, so a less specific name doesn't hurt I think. I'll change it to `dependency_id` unless others are opposed?

Btw, other names I had considered:
* `referenced_object_id`: This is what `mz_object_dependencies` uses and we could use it here too, for consistency. But `mz_object_dependencies` references a bunch of other objects (like functions and type) that this relation doesn't mention, so I didn't want to make it seem like they are too similar.
* `source_id`: We talk about "the sources of data" of a compute object above, so this would fit. But it's easy to confuse with the SQL-level concept of sources (the things in `mz_sources`) which is just a subset of the possible dataflow data sources.